src/main/webapp/components/table/TableExpansionRow.jsx

```diff
@@ -53,7 +54,14 @@ export function getExpansionRow(colSpan, row, moreInfo) {
                         })}
                     </>
                 ) : (
-                    <DL termWidth={250}>{getExpansionRowData(row, moreInfo)}</DL>
+                    <>
+                        <DL termWidth={250}>{getExpansionRowData(row, moreInfo)}</DL>
+                        <Button
+                            to={`/app/search/search?q=search%20index%3D_internal%20source%3D*${row.name}*`}
+                            openInNewContext
+                            label={`Show ${row.name} input events`}
```

Comment:
@hsekowski-splunk I'm not sure how important it is to use internationalization (i18n) for the button label.

Reply:
Also, I will add some CSS later to this button.

Reply:
@tbalar-splunk  ,
i18n added